public/app/features/dashboard/components/PanelEditor/PanelEditor.tsx

```diff
@@ -81,16 +87,47 @@ export class PanelEditor extends PureComponent<Props, State> {
               dashboard={dashboard}
               panel={dirtyPanel}
               isEditing={false}
+              isInEditMode
               isFullscreen={false}
               isInView={true}
             />
           </div>
           <div className={styles.leftPaneData}>
-            <QueriesTab panel={dirtyPanel} dashboard={dashboard} />;
+            <QueriesTab panel={dirtyPanel} dashboard={dashboard} />
           </div>
         </div>
-        <div className={styles.rightPane}>Visualization settings</div>
+        <div className={styles.rightPane}>
+          <Forms.Button
```

Comment:
This is very basic now, needs some more love, like confirmation modal for instance